docs/en/security-privacy-considerations.rst

```diff
@@ -0,0 +1,495 @@
+thus reducing the opportunity for a successful attack. 
+
+.. note::
+  Other security measures are currently under evaluation in issue #117 (https://github.com/italia/eudi-wallet-it-docs/issues/117), 
+  where a list of mitigations from [`OAuthCrossDeviceSec`_] are discussed. Two examples are:
+  
+  - Short Lived/Timebound QR Codes: Reducing the lifetime of the QR code (e.g., 2-3 mins) is fundamental to restrict the time window available for the attacks. 
+  - One-Time QR Codes: One-Time QR codes restrict the possibility of attacks when the same QR code is sent to multiple victims. 
+
+SR-P-41
+~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |check-icon|
+     - The protocol must ensure that an attacker cannot successfully forward an interaction between a Wallet and a Verifier to a Verifier under his own control.
+
+A prerequisite for a successful attack of this kind is that the attacker has access to some messages between the Wallet and the Verifier, for example, 
+the attacker might have access to the presentation contained in the VP Token. Given that, the proper implementation of TLS can avoid the leakage of the response. 
+In addition to TLS, the existing implementation of **nonce** and audience checks in the presentation protocol should help to meet the P-41 security requirement. 
+Regarding the **nonce** claim, the Verifier MUST check that the **nonce** value in the VP Token matches the **nonce** value that is created by the Verifier during the authorization request.
+Regarding the **aud** value, the Verifier must check that the audience of the presentation matches the Verifier's identifier. 
+
+SR-P-50
+~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |partially-check-iconcheck-icon| 
+     - The protocol must ensure that third parties cannot interfere with the binding process.
+
+In the issuance phase, the Holder binding happens at the Credential request to the protected Credential endpoint. This means that the attacker needs to obtain the access token 
+first to be able to call the Credential endpoint and bind the Credentials to the keys under his control. The IT Wallet specification requires the use of a sender-constrained 
+access token, which means that the access token binds to the device using cryptographic materials. 
+
+The second surface for the attack is related to key management. In the case of using software-based keys, it is possible to clone the keys and move them to a device under 
+attacker control, and in the case of stealing the credentials as well, the attacker can easily create proof of possession of the keys. Therefore, it is highly dependent on 
+the secure storage of the keys. In the case of the IT Wallet, it only supports the local internal WSCD that is providing Hardware bound keys that could mitigate the key cloning 
+attack, but it does not provide a high enough level of assurance to meet stringent eIDAS requirements (local internal WSCD has been shown to be insufficiently secure against 
```

Comment:
@grausof ^

Reply:
It would be good to give more details on this analysis and what the references are. Personally I do not agree. I ask for support from @PP-PS

Reply:
We disagree on this part of the Threat Model.

If a sophisticated attacker is able to launch privilege escalation exploits against the OS first, then the Secure World of an ARM TrustZone architecture implementation or extract keys with side channel physical attacks, all bets are off from a practical perspective even with a centralized architecture.

A remote HSM architecture needs a form of WP authentication (fixed tokens, crypto keys for a challenge response auth protocol, etc...) to be permanently stored on the user device (unless we want to force frequent logouts, which are not in scope afaik).

There is no way such authentications tokens for the WP backend on the mobile devices can resist against the same Threat Model.

@peppelinux we request significant changes here.

Reply:
@pp-ps @grausof  I would like to expedite the revision process by incorporating your contributions as suggestions using the tools available in GitHub reviews.

If you're unable to provide your contributions in the form of suggestions, I will need to ask @giadas to interpret your requests, make the necessary changes, and then resubmit them for our review.